frontend/src/components/events/molecules/event_card.css

```diff
@@ -0,0 +1,10 @@
+.eventCard {
+  /* border: 0.5px solid gray; */
+  border-radius: 5px;
+  transition: box-shadow 0.3s;
+  max-width: 400px;
+}
+
+.eventCard:hover {
+  box-shadow: 0 0 11px rgba(33, 33, 33, 0.8);
```

Comment:
hmm this is an awesome idea but a bit intense? maybe turn it down to 5px from 11px 

You can also add a padding of like 5px here and it spaces a bit nicer 

could potentially show multiple looks for the client too